app/views/options.html

```diff
@@ -31,9 +34,45 @@
     </label>
     <p>Example: {{10 * options.velocityUnit().mult|longnum}} meat/{{options.velocityUnit().name}}</p>
   </div>
+  <div class="form-group">
+      <div ng-cloak ng-controller="DropboxdatastoreCtrl">
+get oauth key from <a href="https://dbxoauth2.site44.com/">https://dbxoauth2.site44.com/</a><br>
+use appid: {{app_key}} and paste into  this box
+ <!--input  ng-model="app_key" 
+currently authorised???: {{isAuth()}} <br>
+-->
+ <form id="savegame-form" ng-submit="doAuth()">
+ <input  ng-model="oauth_token" ng-change"doAuth()" >
+</form>
+<br>
+<br>
+
+ <form id="savegame-form" ng-submit="addSavegame()">
+type name and press enter to save
```

Comment:
ahh yes.. the first get tables fails if it doesnt exists..  so thats just an initialization issue